BossMod/Components/Knockback.cs

```diff
@@ -150,6 +154,12 @@ protected IEnumerable<(WPos from, WPos to)> CalculateMovements(BossModule module
                     break;
             }
         }
+
+        internal void OnCastFinished(BossModule module, Actor caster, ActorCastEvent spell)
+        {
+            throw new NotImplementedException();
```

Comment:
Why? This is not a good idea, knockback components shouldn't require overriding OnCastFinished

Reply:
I don't know, I haven't added this. VS Code must have added it on its own